python/pyspark/tests.py

```diff
@@ -179,9 +179,12 @@ def test_in_memory_sort(self):
                          list(sorter.sorted(l, key=lambda x: -x, reverse=True)))
 
     def test_external_sort(self):
+        class CustomizedSorter(ExternalSorter):
+            def _next_limit(self):
```

Comment:
It seems like the intent here is to mock `get_used_memory()`; I'm not super familiar with Python mocking frameworks, but if this was Java then I imagine that `get_used_memory()` would be a method of some context / environment object that we could mock.